Our Scripts/MPhys_model.py

```diff
@@ -102,19 +102,17 @@ def P_L_Lya(z): # luminosity density of lyman alpha
     returns P_L_Lya
     """
 
-    x = pylab.array([2.2,2.5,2.8,3.0,3.2, 3.3, 3.7, 4.1, 4.6, 4.8, 5.1, 5.3, 5.8,7,10,11,12,13,14,16])
-    y1 = pylab.array([0.52, 0.74, 0.77, 0.88, 0.84, 0.85, 1.01, 0.87, 1.19, 1.12, 1.27, 1.08, 1.10,.7,0.5,0.3,0.10,0.05,0.001,0.001])  # data from SC4K Sobral 
+    x = pylab.array([2.2,2.5,2.8,3.0,3.2, 3.3, 3.7, 4.1, 4.6, 4.8, 5.1, 5.3, 5.8])#,7,10,11,12,13,14,16])
+    y1 = pylab.array([ 0.52, 0.74, 0.77, 0.88, 0.84, 0.85, 1.01, 0.87, 1.19, 1.12, 1.27, 1.08, 1.10]) #,.7,0.5,0.3,0.10,0.05,0.001,0.001])  # data from SC4K Sobral 
     y = [math.log10(i*10**40) for i in y1]
-    y_sigma = pylab.array([0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01])
-    p2 = pylab.polyfit(x, y, 3.0)
+    y_sigma = pylab.array([0.05,0.075,0.095, 0.095,0.09,0.095,0.18,0.13,0.35,0.32,0.40, 0.185, 0.185])#,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01])
+    p2 = pylab.polyfit(x, y, 2.0)
```

Comment:
Different data sets here, which one is the most up to date?

Reply:
the y_sigma with values other than 0.01 is the correct one (it was just a place holder)